lstchain/tools/lstchain_create_dl3_file.py

```diff
@@ -103,6 +107,11 @@ class DataReductionFITSWriter(Tool):
         help="DEC position of the source"
     ).tag(config=True)
 
+    optimize_gh = traits.Bool(
```

Comment:
But this is *applying* the cuts, not calculating them. Why is a switch needed at all?

I think the DL3 tool should just look into the IRF file and if it finds the `GH_CUTS` table, apply the energy dependent cuts.

Do we also store the non-genergy-dependent cut in the IRF file as header? We should, and then this script can also use that information in case of a global cut.

Reply:
I think, letting the user have the option to choose, whether to use energy-dependent or global cuts while creating DL3 Tool is good. 

The IRF Tool does add the header value of global `GH_CUT` and `RAD_MAX` when used.
If the user-provided IRF does not contain the `GH_CUTS` table, or a header value of the global `GH CUT` applied, then the Tool will raise an error.

I have not included a check on whether the IRF has the global `GH_CUT` to use by default, global gammaness cut when creating the DL3 file. I will fix that.

Reply:
> I think, letting the user have the option to choose, whether to use energy-dependent or global cuts while creating DL3 Tool is good.

But that's not possible here, since the IRF file has to have the same settings, otherwise you create and invalid combination of `EVENTS` table with the IRFs. 

The user can only choose when creating the IRFs, not when creating the DL3 Events file. At this point, the same settings as used for the IRF calculation **must** be applied.

Reply:
Ok. I will make the changes accordingly.